package.json

```diff
@@ -107,7 +107,8 @@
     "sass-loader": "^8.0.2",
     "source-map-loader": "^1.1.3",
     "style-loader": "^1.3.0",
-    "webpack-cli": "^4.7.0"
+    "webpack-cli": "^4.7.0",
+    "yamljs": "^0.3.0"
```

Comment:
this seems not used anywhere. there is already a module called `js-yaml` being imported.

Reply:
will switch to `js-yaml`